doc/tut1.rst

```diff
@@ -824,7 +824,7 @@ Forward declarations
 
 Every variable, procedure, etc. needs to be declared before it can be used.
 (The reason for this is that it is non-trivial to avoid this need in a
-language that supports meta programming as extensively as Nim does.)
+language that supports meta-programming as extensively as Nim does.)
```

Comment:
```suggestion
language that supports metaprogramming as extensively as Nim does.)
```

I think that in most cases Nim docs/tutorials/blog posts refer to metaprogramming without any hyphens

Reply:
I had my doubts about this one as well, tbh...